src/assets/js/components/event-container.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="event-container">
+        <h4>{{ text }}</h4>
+        <template v-for="message in messages">
+            <event-message v-bind:key="message.index" v-bind:message="message"/>
+        </template>
+    </div>
+</template>
+
+<script>
+import EventMessage from './event-message.vue';
+
+export default {
+    props: ['text'],
+    data(){
+        return{
+            messages: getData()
+        }
+    },
+    components:{
+        EventMessage
+    }
+}
+
+ function getData(url){
```

Comment:
Nešlo by tohle udělat nějak abstraktněji ? Resp. nedávalo by větší smysl předávat tyto data do komponenty z venku a je to univerzál, pak  to můžeš udělat, jak bude potřeba třeba asynchronně, jak jsem psal v emailu ?

Reply:
Rozumím...Nicméně stále to není obecně použitelný, představ si, že si vemu tvoji komponentu a budu si chtít sám definovat logiku pro získání dat, tedy nestačilo by si ty  data předat z venku přes props ?